server.js

```diff
@@ -0,0 +1,77 @@
+import { Application } from "https://deno.land/x/abc/mod.ts";
+import { DB } from "https://deno.land/x/sqlite@v2.5.0/mod.ts";
+import { abcCors } from "https://deno.land/x/cors/mod.ts";
+import * as bcrypt from "https://deno.land/x/bcrypt/mod.ts";
+import { v4 } from "https://deno.land/std/uuid/mod.ts";
+
+const db = new DB("users.db");
+const app = new Application();
+const PORT = 8080;
+
+const corsConfig = abcCors({
+  origin: "*",
+  allowedHeaders: ["Authorization", "Content-Type", "Accept", "Origin", "User-Agent"],
+  credentials: true,
+});
+
+app
+  .use(corsConfig)
+  .post("/login", (server) => postLogin(server))
+  .post("/createaccount", (server) => postAccount(server))
+  .start({ port: PORT });
+
+console.log(`Server running on http://localhost:${PORT}`);
+
+async function postLogin(server) {
+  try {
+    const { username, password } = await server.body;
+    if (!username || !password) {
+      return server.json({ success: false, error: "Need to include a username and password" }, 400);
+    }
+    // Get the users password stored in the database.
+    const [response] = [
+      ...(await db
+        .query("SELECT id, username, password_encrypted FROM users WHERE username = ?", [username])
```

Comment:
Response is selecting id, username and psssword_encrypted from users but i think we only need to select id and password_encrypted.